apps/src/dance/ai/dance-ai-modal.module.scss

```diff
@@ -299,5 +342,11 @@
         font-size: 16px;
       }
     }
+
+    .buttonsAreaTopRight {
```

Comment:
This feels not great...acknowledging that all of the UI specifics are very in flux, but this feels like the type of thing that wouldn't be fixed as designs get updated? Is there a more semantic HTML container to put the button in that would make this feel less odd (looks like its not in the same container as the new block preview header, where I'd expect it)?

Same comment re: lots of !important being thrown in the `.dialog` styling, do we have any options there (or is the styling so far from done that it doesn't feel worthwhile to get right)?

Reply:
Good call.  I had hoped to keep all the buttons together, but for now have moved these two buttons to the header.

I'd love to get rid of the `!important` tags, but they'd require a change in how the shared `AccessibleDialog` component works.  There's an interesting internal thread about it [here](https://codedotorg.slack.com/archives/C03DBDN67B7/p1694201912817799).  I think that would be best done separately to this change.

Reply:
Thanks for the context re: `!important`. Moving the button into the header looks like a nice improvement to me.